conference.js

```diff
@@ -970,7 +971,7 @@ export default {
                 this.audioMuted = false;
             }
 
-            APP.UI.enableMicrophoneButton();
+            APP.UI.disableMicrophoneButton(false);
```

Comment:
I realize you didn't introduce it but, since you modified it, can we not use some "standard" method naming like you used for `_setToolbarButtonToggled(boolean)` or like we have `setAudioMuted(boolean)`? For example, `setMicrophoneButtonEnabled(boolean)`? Apart from sounding non-"standard" to me, `disableMicrophoneButton(false)` to me rings somewhat like a double negation...